frontend/pages/_document.tsx

```diff
@@ -12,10 +12,6 @@ class MyDocument extends Document {
                   href="https://fonts.gstatic.com"
                   crossOrigin="anonymous"
                />
```

Comment:
Do we still want to preconnect to the google font api?

Reply:
This too was added last month or so. So yep a little cleanup is needed.